app/sources/colombia.py

```diff
@@ -123,6 +123,8 @@ def get_new_contracts(index: int, from_date: datetime, until_date: datetime | No
     )
 
     if from_date and until_date:
+        until_date = until_date.strftime(date_format)
+        from_date = from_date.strftime(date_format)
```

Comment:
This should be in `fetch_all_awards_from_period` instead, because this function expects datetimes.

Reply:
@yolile